sql/load_demo_data.sql

```diff
@@ -1089,7 +1125,11 @@ CREATE TABLE `oh_dicom` (
   `DM_ACTIVE` tinyint(1) NOT NULL DEFAULT 1,
```

Comment:
This table sill has `DM_DATA` defined as one of the columns; it doesn't exist any where else that I can find in the code.

Reply:
removed, thanks